docs/setup/activate-conda-env.md

```diff
@@ -0,0 +1,96 @@
+# Step 6: Activate the Conda Environment for `assign0`
+
+[Getting Started as a Student](./getting-started.md)
+
+[previous: Conda Installation](./conda-install.md)
```

Comment:
maybe there is better formatting for this? idk. Not the best front-end designer 😂